inst/mappings/mapping_ECEMF.csv

```diff
@@ -311,6 +311,10 @@ Energy Service|Transportation|Passenger|LDV;Gpkm/yr;;;;;;;
 Energy Service|Transportation|Passenger|Rail;Gpkm/yr;;;;;;;
 Energy Service|Transportation|Passenger|Truck;Gpkm/yr;;;;;;;
 Energy System Cost;billion EUR_2020/yr;Energy system costs;billion US$2017/yr;0.9502;;;;R
+Energy System Cost|Supply;billion EUR_2020/yr;Energy system costs;billion US$2017/yr;0.9502;;;;R
+Energy System Cost|Supply|Fuel Cost;billion EUR_2020/yr;Energy System Cost|Supply|Fuel Cost;billion US$2017/yr;0.9502;;;;R
+Energy System Cost|Supply|Investments;billion EUR_2020/yr;Energy System Cost|Supply|Investments;billion US$2017/yr;0.9502;;;;R
+Energy System Cost|Supply|Operation and Maintenance Cost;billion EUR_2020/yr;Energy System Cost|Supply|Operation and Maintenance Cost;billion US$2017/yr;0.9502;;;;R
 Energy System Cost|Power Sector;billion EUR_2020/yr;;;;;;;
 Energy to Power Ratio|Electricity|Storage|Battery;h;;;;;;;
```

Comment:
random side questin: is it possible to simply report a number here? (in this case 6)?